data/strings/types_strings.txt

```diff
@@ -30492,10 +30515,13 @@
     zh-Hant = 視野
 
   [type.tourism.wilderness_hut]
+    comment = Typically more basic, not staffed and free (compared to alpine_hut).
     en = Wilderness Hut
```

Comment:
add `he = בקתה לא מאוישת` (lit. unmanned hut; I couldn't find a proper term for this)
alternatively just 'hut' `he = בקתה`

Reply:
thanks!